packages/flutter_driver/lib/src/driver/timeline_summary.dart

```diff
@@ -159,6 +168,19 @@ class TimelineSummary {
     return total / durations.length;
   }
 
+  double _percentileInMillis(Iterable<Duration> durations, double percentile) {
+    if (durations.isEmpty)
+      return null;
+
+    assert(percentile >= 0 && percentile <= 100);
```

Comment:
To be more precise, let's compare against `0.0` and `100.0`. In Dart `0` and `100` are integers, and it's not immediately obvious in which direction type coercions are happening.